pkg/entrypoint/entrypointer.go

```diff
@@ -18,12 +18,13 @@ package entrypoint
 
 import (
 	"fmt"
-	"go.uber.org/zap"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"time"
 
+	"go.uber.org/zap"
+
```

Comment:
odd spacing choice.  do we separate like this anywhere else in the codebase?

Reply:
Yeah the preferred style is to keep stdlib imports separate.

Reply:
I meant the separation between zap and our internal packages.  We don't do that anywhere else.

Reply:
Oh yeah. Some people do that, some don't. I don't really care except I make it a policy never to disagree with formatting tools. 🤖

Reply:
it's over.  the robots have won.